packages/shared/lib/services/sync/config/deploy.service.ts

```diff
@@ -268,33 +269,41 @@ export async function upgradePreBuilt({
     delete flowData.id;
 
     try {
-        const [syncId] = await db.knex.from<SyncConfig>(TABLE).insert(flowData).returning('id');
+        const [newSyncConfig] = await db.knex.from<SyncConfig>(TABLE).insert(flowData).returning('id');
 
         const endpoints: SyncEndpoint[] = [];
-        if (flow.endpoints) {
-            flow.endpoints.forEach((endpoint, endpointIndex) => {
-                const method = Object.keys(endpoint)[0] as HTTP_VERB;
-                const path = endpoint[method] as string;
-                const res: SyncEndpoint = {
-                    sync_config_id: syncId?.id as number,
-                    method,
-                    path,
-                    created_at: now,
-                    updated_at: now
-                };
-                const model = flowData.models[endpointIndex] as string;
-                if (model) {
-                    res.model = model;
-                }
-                endpoints.push(res);
-            });
-        }
 
-        if (endpoints.length > 0) {
-            await db.knex.from<SyncEndpoint>(ENDPOINT_TABLE).insert(endpoints);
-        }
+        const newSyncConfigId = newSyncConfig?.id;
+        if (newSyncConfigId) {
```

Comment:
with this if we go straight to `logCtx.success()` which seems incorrect
I would do `if (!newSyncConfig.id) throw new Error()`

Reply:
that was the behavior before but yeah I agree that doesn't seem right. will fix